src/tools/rustfmt/src/items.rs

```diff
@@ -1944,6 +1944,11 @@ pub(crate) fn rewrite_struct_field(
     shape: Shape,
     lhs_max_width: usize,
 ) -> RewriteResult {
+    // FIXME(default_field_values): Implement formatting.
```

Comment:
This also should go above the `contains_skip` since that would mess up:

```
struct Foo2 {
    #[rustfmt::skip]
    default_field:    Spacing =    /* uwu */ 0,
}
```

which I added a test for.